devenv/setup.sh

```diff
@@ -14,6 +14,20 @@ bulkDashboard() {
 		ln -s -f -r ./bulk-dashboards/bulk-dashboards.yaml ../conf/provisioning/dashboards/custom.yaml
 }
 
+bulkAlertingDashboard() {
+
+		requiresJsonnet
+
+		COUNTER=0
+		MAX=100
+		while [  $COUNTER -lt $MAX ]; do
+				jsonnet -o "bulk_alerting_dashboards/alerting_dashboard${COUNTER}.json" -e "local bulkDash = import 'bulk_alerting_dashboards/bulkdash_alerting.jsonnet'; bulkDash + {  uid: 'uid-${COUNTER}',  title: 'title-${COUNTER}' }"
```

Comment:
Think these uid's will be the same as when generating bulkDashboard() above. WIll try and fix that